include/context_menu.js

```diff
@@ -1,5 +1,13 @@
+var tabId = false;
+
 function contextMenuOnClick(info, t) {
-    chrome.tabs.create({url: "main.html#" + info.selectionText});
+    if (tabId === false) {
+      chrome.tabs.create({url: "main.html#" + info.selectionText}, function(tab) {
+        tabId = tab.id;
+      });
+    } else {
```

Comment:
I don’t follow a particular coding style. But you can refer to other js files. Thanks!